config.md

```diff
@@ -265,6 +265,19 @@ Annotations are key-value maps.
 }
 ```
 
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
+
+## Kernel-configuration
+
+Used by VM-based runtimes only.
+
+* **`path`** (string, required) specifies an absolute path to the kernel
+  used to boot the virtual machine.
+* **`parameters`** (string, optional) specifies a space-separated list of parameters to pass to the kernel.
+* **`initrd`** (string, optional) specifies an absolute path to an initial ramdisk to be used by the kernel.
```

Comment:
On Wed, May 11, 2016 at 10:34:58AM -0700, Geronimo Orozco wrote:
> > +* **`path`** (string, required) specifies an absolute path to the kernel
> > +  used to boot the virtual machine.
> > +* **`parameters`** (string, optional) specifies a space-separated list of parameters to pass to the kernel.
> > +* **`initrd`** (string, optional) specifies an absolute path to an initial ramdisk to be used by the kernel.
> 
> What's the problem with absolute paths? 

I'm fine allowing absolute paths, but I don't see why we shouldn't
allow relative paths too [1].

[1]: https://github.com/opencontainers/runtime-spec/pull/405/files/8d359458aef03c76837ed17eb3fcfb935f0708b2#r62714446